parts/linux/cloud-init/artifacts/cgroup-stats.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Update cgroup statisitics to Kusto on a 5 minute timer
+
+[Timer]
+OnBootSec=0min
+OnCalendar=*:0/5
```

Comment:
I think this is every 5 sec. I think you want
`*-*-* *:0/5:0` or similar. try it out?

`systemctl list-timers` to see invocation time and next schedule